src/main/webapp/gast/layout/footer.inc.jsp

```diff
@@ -1,34 +1,82 @@
 <%@ page import="de.uni_tuebingen.ub.nppm.util.Utils" isThreadSafe="false" %>
-<% //close <div id="page"> and <div id="page-wrap">, see header.inc.jsp  %>
-  </div>
-</div>
-<div id="footer">
-    <div id="footer-menu">
-        <p> <a href="<%=Utils.getBaseUrl(request)%>/gast/startseite"> NeG </a>|
-         <a href="http://www.neg.uni-tuebingen.de/?q=de/quellenliste"> Quellenliste </a>|
-         <a href="http://www.neg.uni-tuebingen.de/?q=de/tagungen"> Tagungen </a>|
-         <a href="http://www.neg.uni-tuebingen.de/?q=de/mitglieder"> Mitglieder </a>|
-         <a href="http://www.neg.uni-tuebingen.de/?q=de/projekte"> Projekte </a>|
-         <a href="http://www.neg.uni-tuebingen.de/?q=de/publikationen"> Publikationen </a>|
-         <a href="http://www.neg.uni-tuebingen.de/?q=de/ziele"> Ziele </a>|
-        <a href="<%=Utils.getBaseUrl(request)%>/gast/impressum"> Impressum </a>
-        </p>
+<footer id="ut-identifier--footer" class="ut-page__footer ut-page-footer">
```

Comment:
If you look at the footer at a small resolution (e.g. <=767), the panels in the footer will collapse, and you will not be able to expand them again. I am not sure whether this is a class-related problem or a JS related problem.